cmd/kube-scheduler/app/options/options.go

```diff
@@ -297,6 +297,9 @@ func (o *Options) Config() (*schedulerappconfig.Config, error) {
 	dynClient := dynamic.NewForConfigOrDie(kubeConfig)
 	c.DynInformerFactory = dynamicinformer.NewFilteredDynamicSharedInformerFactory(dynClient, 0, corev1.NamespaceAll, nil)
 	c.LeaderElection = leaderElectionConfig
+	if o.Deprecated != nil {
```

Comment:
Shouldn't we move this to here, as `--config` not specified. https://github.com/kubernetes/kubernetes/blob/2047936f3f8ddd96498dcc812631b5368d14f3be/cmd/kube-scheduler/app/options/options.go#L110-L141

Reply:
It is just used to set to CC。